### ISSUE TITLE
Duplicate properties issue on "transition-property"

### ISSUE THREAD

**Issue by Mushr0000m** [postcss/autoprefixer#602]
Hi,

I've the following code : 
```
#nav-toggle span::before {
    transition-property: top, transform;
}
```
But for some reason AutoPrefixer give me (with default last 2 versions) : 
```
#nav-toggle span::before {
   transition-property: top, -webkit-transform;
   transition-property: top, transform;
   transition-property: top, transform, -webkit-transform;
}
```
The last line with "-webkit-transform;" break the css rule on Safari because it's a duplicate property. I've tried on https://autoprefixer.github.io/ and same result. Any idea to fix this ?

Thanks

**Comment by ai** [comment]
Can you show me this rule inside Safari DevTools? What rules it ignore.

**Comment by Mushr0000m** [comment]
<img width="237" alt="capture d ecran 2016-01-04 a 20 03 03" src="https://cloud.githubusercontent.com/assets/5532323/12097824/4507c2e2-b31e-11e5-9468-92418ced5a71.png">
It ignore the first two ones and so it's the last one that is used. The last one containing "transform" and "-webkit-transform" it breaks.

**Comment by ai** [comment]
So, what exactly problem with last `transition-property: top, transform, -webkit-transform`.

Safari should take `transform` and old `-webkit-transform` both.

**Comment by Mushr0000m** [comment]
The problem is that I have declared 2 properties in `transition-property`that are `top` and `transform`. So in combination of that I have declared an other css rule for some animation : 
```
#nav-toggle span.active::before {
    top: 0;
    transition-delay: 0ms, 300ms;
    transition-duration: 300ms, 300ms;
}
```
So Safari read the last line of the css prefixed that contains `transition-property: top, transform, -webkit-transform`. As you can see I have 3 properties here but only declared 2 in `transition-delay` and `transition-duration`.

So to sum up if you declare 2 transitions properties / durations / delays and end up with 3 transitions properties the animation breaks. I'm not sure I was clear ^^ Do you see the issue ?

**Comment by ai** [comment]
Yeap, this is why Autoprefixer print you a special warning:

```
Replace transition-property to transition, because Autoprefixer could not support  any cases of transition-property and other transition-*
```

Sorry, but it is a oonly way how we can prefix transition for all browsers. You can look for issues with others way ;).

**Comment by Mushr0000m** [comment]
OK I'll have a look at that. But one last question, I don't have any warnings in the terminal, I can't find in the doc how to activated them, I'm using gulp-autoprefixer.

**Comment by ai** [comment]
@Mushr0000m you shold move to `gulp-postcss` https://github.com/postcss/autoprefixer#gulp
